UPGRADE-API-2.1.md

```
@@ -1,3 +1,17 @@
+# UPGRADE FROM `2.1.10` TO `2.1.11`
+
+### Constructor signature changes
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)